node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java

```diff
@@ -24,7 +24,7 @@
     private static final Logger logger = Logger.getLogger(Editor.class.getName());
     private static final Charset ENCODING = StandardCharsets.UTF_8;
 
-    private static int maxLength = 300;
+    private static final int MAX_LENGTH = 300;
```

Comment:
From some of the diffs I have seen - I think we can raise this to say 500?

Reply:
Changed to 500